apps/06.1Checkers.js

```diff
@@ -10,125 +10,126 @@ var rl = readline.createInterface({
     this.viewGrid = function () {
-// add our column numbers
-    var string = "  0 1 2 3 4 5 6 7\n";
-    for (var row = 0; row < 8; row++) {
- // we start with our row number in our array
-    var rowOfCheckers = [row];
-// a loop within a loop
-    for (var column = 0; column < 8; column++) {
-// if the location is "truthy" (contains a checker piece, in this case)
-    if (this.grid[row][column]) {
+        // add our column numbers
+        var string = "  0 1 2 3 4 5 6 7\n";
+        for (var row = 0; row < 8; row++) {
+            // we start with our row number in our array
+            var rowOfCheckers = [row];
+            // a loop within a loop
+            for (var column = 0; column < 8; column++) {
+                // if the location is "truthy" (contains a checker piece, in this case)
+                if (this.grid[row][column]) {
                     // push the symbol of the check in that location into the array
-    rowOfCheckers.push(this.grid[row][column].symbol);
-}   else {
+                    rowOfCheckers.push(this.grid[row][column].symbol);
+                } else {
                     // just push in a blank space
-    rowOfCheckers.push(' ');
-}
-}
+                    rowOfCheckers.push(' ');
+                }
+            }
             // join the rowOfCheckers array to a string, separated by a space
-    string += rowOfCheckers.join(' ');
+            string += rowOfCheckers.join(' ');
             // add a 'new line'
-    string += "\n";
-}
-    console.log(string);
-};
+            string += "\n";
+        }
+        console.log(string);
+    };
 
     //create the array thats going to hold the coordinates of checkers
     this.checkers = [];
     this.createCheckers = function () {
             //create two arrays for initial positions of black and white checkers
-    var whitePositions = [];
-    var blackPositions = [];
+            var whitePositions = [];
+            var blackPositions = [];
             //fill the two initial positions arrays with the coordinates of the checkers 
-    var j = 0;
-    while (j < 3) {
-    for (var i = 1; i <= 7; i = i + 2) {
-    whitePositions[j] = [0, i];
-    whitePositions[j + 8] = [2, i];
-    blackPositions[j + 4] = [6, i];
-    j++;
-}
-    j = 0;
-    for (var i = 0; i < 7; i = i + 2) {
-    whitePositions[j + 4] = [1, i];
-    blackPositions[j] = [5, i];
-    blackPositions[j + 8] = [7, i];
-    j++;
-}
-}
+            var j = 0;
+            while (j < 3) {
+                for (var i = 1; i <= 7; i = i + 2) {
+                    whitePositions[j] = [0, i];
+                    whitePositions[j + 8] = [2, i];
+                    blackPositions[j + 4] = [6, i];
+                    j++;
+                }
+                j = 0;
+                for (var i = 0; i < 7; i = i + 2) {
+                    whitePositions[j + 4] = [1, i];
+                    blackPositions[j] = [5, i];
+                    blackPositions[j + 8] = [7, i];
```

Comment:
Nice for loops but since the initial whitePositions and blackPositions are static and not subject to user input, you can simply hard code them inside their parent variables.

ex:
`var blackPosition = [[5, 0], [5, 2], [5, 4], [5, 6],...`